src/CMakeLists.txt

```diff
@@ -547,10 +547,21 @@ endif(HAVE_ARMV8_CRC)
 
 add_library(common_utf8 STATIC common/utf8.c)
 
+if(${WITH_LIBZBC})
+	message("Adding LIBZBC support")
+  add_subdirectory(libzbc)
+endif(${WITH_LIBZBC})
+
 if(${WITH_LTTNG})
   add_subdirectory(tracing)
 endif(${WITH_LTTNG})
 
+if(${WITH_LIBZBC})
+  set(LIBZBC_INCLUDE_PATH,libzbc/include)
+  include_directories(${LIBZBC_INCLUDE_PATH})
```

Comment:
better off using `target_include_directories()` if libzbc is only used by a target.